src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,58 @@
+package lotto.view;
+
+import java.text.DecimalFormat;
+import java.util.Arrays;
+import lotto.domain.LottoResult;
+import lotto.domain.LottoTicket;
+import lotto.domain.Profit;
+import lotto.enums.Rank;
+
+public class OutputView {
+    private static final String RATE_FORMAT = "#,##0.0";
+
+    private OutputView() {
+    }
+
+    public static void printPurchaseInputText() {
+        System.out.println("구입금액을 입력해 주세요.");
+    }
+
+    public static void printPurchasedCount(LottoTicket lottoTicket) {
+        System.out.printf("%d개를 구매했습니다.", lottoTicket.getLottoCount());
+        printNewLine();
+    }
+
+    public static void printPurchasedLottoTicket(LottoTicket lottoTicket) {
+        StringBuilder lottoPrinter = new StringBuilder();
+        lottoTicket.getLottoTicket().stream()
+                .forEach(lotto -> lottoPrinter.append(lotto).append("\n"));
```

Comment:
이 부분은 스트림을 쓰지 않고 forEach 람다만으로도 가능할 것 같습니다!
```suggestion
        lottoTicket.getLottoTicket()
                .forEach(lotto -> lottoPrinter.append(lotto).append("\n"));
```